### DISCUSSION TITLE
Mealie LDAP

### DISCUSSION THREAD

**Discussion by motermouth15** [mealie-recipes/mealie#887] [Q&A]
Hey there,

I recently setup my own instance of Mealie via docker. I love it so far but wanted a centralized login, so I also set up a new OpenLDAP instance via docker as well. I've successfully configured mealie so that LDAP members can authenticate.

I also set up a 'groupofmembers' in my LDAP tree, and am wondering if there's a way within Mealie to only allow members of that specific group to successfully authenticate?

I appreciate the help, thank you!
-Nate

**Comment by michael-genson** [comment]
Please see the LDAP and backend docs: https://docs.mealie.io/documentation/getting-started/authentication/ldap/

I'm not 100% sure this is doable with LDAP, but it definitely is with [OIDC](https://docs.mealie.io/documentation/getting-started/authentication/oidc/)
